### ISSUE TITLE
[Zero silent failures] Facility locator / facilities-api

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#19249]
## Acceptance Criteria
- [ ] Testable_Outcome_X
- [ ] Testable_Outcome_Y
- [ ] Testable_Outcome_Z
- [ ] Requires design review
- [ ] Requires accessibility review

**Comment by eselkin** [comment]
The prime candidates for needing more error notification are:
If PPMS is down and someone selects "Community Providers in VA's network" 
They will not be notified that the "Service type" will not be populated with proper data so even if they enter data it will complain that they are not selecting something even though it's the system not providing options for them to select.

**Comment by eselkin** [comment]
I created this issue: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/19332

**Comment by jilladams** [comment]
@eselkin please port your user diagram to the [facilities-api docs](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/facilities/facilities-api), if you haven't already. 

**@Agile6MSkinner & @mmiddaugh ** recapping where we are so we can see about closing this ticket: 
* Eli found one silent failure: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/19249#issuecomment-2369617494
* He created #19332 to resolve it
  * In sprint planning: we turned that into a UX ticket
  * When Dave is done, it will need a follow on engineering ticket, to implement the UX
* There is one open question in the checklist: 

> Has the owner of the system of record receiving the user's data indicated in writing that their system notifies or resolves 100% of fatal errors once in their custody?
> If not, work with OCTO to meet with the owner of the system and get their agreement in writing.
> Please document the outcome of this conversation in your product's documentation in Github.

**Eli's answer:** I do not know if we have such an understanding with PPMS or Lighthouse

In this case: we aren't submitting applications that go into a void. We are submitting search parameters that are either successful and show search terms, or are unsuccessful and we show an error to the user. I think that's sufficient for the use case and am unsure what LH or PPMS could do differently re: "system of record receiving the user's data indicated in writing that their system notifies or resolves 100% of fatal errors once in their custody". Do you all agree? IF so, we can close. If not, we'll need to follow up with LH / PPMS and help them understand what else we need them to do for us.

**Comment by Agile6MSkinner** [comment]
@jilladams Looking at Dave designs, the ticket we have will suffice to fix this as a silent failure. Considering how much extra work it would take to chase this all the way down the rabbit hole, including relying on other teams, and that it wouldn't necessarily result in anything resembling an iron-clad fix, the work we have ticketed allows us to close this out.

**Comment by jilladams** [comment]
OK! @eselkin once the diagram is saved out into docs, feel free to close!

**Comment by eselkin** [comment]
Does it belong in facilities-api or facility-locator docs? It seems more facility-locator rather than api, but it does hinge on the data

**Comment by jilladams** [comment]
Yeah, I think it starts with the user at Facility Locator, so probably should go there. We can link back to facilities-api docs as needed, if we can't find it in the future.

**Comment by eselkin** [comment]
I put the chart here:
https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/facilities/facility-locator/engineering/flow_diagram_facility_locator_errors.md
